libs/cardano-ledger-core/src/Cardano/Ledger/CertState.hs

```diff
@@ -312,6 +313,14 @@ data VState era = VState
        )
   , vsDRepDistr :: !(DRepDistr (EraCrypto era))
   , vsCommitteeState :: !(CommitteeState era)
+  -- | Number of contiguous epochs in which there are exactly zero
+  -- active governance proposals to vote on. It is incremented in every
+  -- EPOCH rule if the number of active governance proposals to It
+  -- is reset to zero when a new governance action is successfully
+  -- proposed. vote on continues to be zero.
```

Comment:
maybe `vote on continues to be zero.`  this is the continuation of the sentence from before :laughing: